### PR TITLE
feature #329- reset the user

### PR DESCRIPTION
**Pull request by ketan9190**
New Feature

feature #329- reset the user in account manager

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1968,6 +1968,53 @@ program
...
+                if (answers['ok']) {
+                    resetUser(login, asJson);
+                }
+            });
+        }
+    }).on('--help', function () {
+        console.log('');
+        console.log('  Details:');
+        console.log();
+        console.log('  Reset a user.');
```
Comment (tobiaslohr) [inline cli.js:2007]: Can you please add some more details here on what the rest does? My suggestion:

> Reset a user. This is useful when the user forgot their password or when you want to unlink the Account Manager user account from a Salesforce account. An email is automatically sent to the user's email address with an activation link.
> This requires permissions ...

**Review by tobiaslohr — CHANGES_REQUESTED** [review]
Thanks a lot @ketan9190 for contributing! Can you please check the change requested below?

File: cli.js (hunk 1 above)
Comment (ketan9190) [reply cli.js]: Added @tobiaslohr

**Comment by ketan9190** [comment]
> Thanks again for your contribution!

Welcome @tobiaslohr !!

**Comment by tobiaslohr** [comment]
Fixes #329

**Pull request merged**